[2023-11-10T00:36:40.494+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2023-11-09T00:00:00+00:00 [queued]>
[2023-11-10T00:36:40.499+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2023-11-09T00:00:00+00:00 [queued]>
[2023-11-10T00:36:40.499+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-11-10T00:36:40.499+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-11-10T00:36:40.499+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-11-10T00:36:40.505+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): store_user> on 2023-11-09 00:00:00+00:00
[2023-11-10T00:36:40.510+0000] {standard_task_runner.py:55} INFO - Started process 19600 to run task
[2023-11-10T00:36:40.512+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'scheduled__2023-11-09T00:00:00+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpl5byjy24']
[2023-11-10T00:36:40.512+0000] {standard_task_runner.py:83} INFO - Job 69: Subtask store_user
[2023-11-10T00:36:40.543+0000] {task_command.py:388} INFO - Running <TaskInstance: user_processing.store_user scheduled__2023-11-09T00:00:00+00:00 [running]> on host cff42873677f
[2023-11-10T00:36:40.573+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2023-11-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-11-09T00:00:00+00:00
[2023-11-10T00:36:40.574+0000] {postgres.py:158} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2023-11-10T00:36:40.578+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-11-10T00:36:40.581+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-11-10T00:36:40.585+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20231109T000000, start_date=20231110T003640, end_date=20231110T003640
[2023-11-10T00:36:40.608+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-11-10T00:36:40.617+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
